setup.py

```diff
@@ -1,5 +1,8 @@
 from setuptools import setup, find_packages
 
+with open("fanova/__version__.py") as fh:
+    version = fh.readlines()[-1].split()[-1].strip("\"'")
+
 setup(
     name = "fanova",
     version = "2.0.3",
```

Comment:
Then the Version should also be set to the __version__ content
```suggestion
    version = version,
```

Reply:
*chrm* thanks.